frontend-service/src/test/java/com/amazonaws/blox/frontend/integration/IntegrationTestBase.java

```diff
@@ -40,28 +36,9 @@
 )
 public abstract class IntegrationTestBase {
```

Comment:
Now that we'll only have the one integration test, is it still worth having the base class?

Reply:
I plan to use this one for exception testing as well.  Will evaluate if it's still needed then.